tests/js/server/dump/dump-mixed.js

```diff
@@ -0,0 +1,553 @@
+/*jshint globalstrict:false, strict:false, maxlen:4000 */
+/*global assertEqual, assertNotEqual, assertTrue, assertFalse, assertNull */
+
+////////////////////////////////////////////////////////////////////////////////
+/// @brief tests for dump/reload
+///
+/// @file
+///
+/// DISCLAIMER
+///
+/// Copyright 2010-2012 triagens GmbH, Cologne, Germany
+///
+/// Licensed under the Apache License, Version 2.0 (the "License");
+/// you may not use this file except in compliance with the License.
+/// You may obtain a copy of the License at
+///
+///     http://www.apache.org/licenses/LICENSE-2.0
+///
+/// Unless required by applicable law or agreed to in writing, software
+/// distributed under the License is distributed on an "AS IS" BASIS,
+/// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+/// See the License for the specific language governing permissions and
+/// limitations under the License.
+///
+/// Copyright holder is triAGENS GmbH, Cologne, Germany
+///
+/// @author Jan Steemann
+/// @author Copyright 2012, triAGENS GmbH, Cologne, Germany
+////////////////////////////////////////////////////////////////////////////////
+
+var internal = require("internal");
+var jsunity = require("jsunity");
+var analyzers = require("@arangodb/analyzers");
+
+////////////////////////////////////////////////////////////////////////////////
+/// @brief test suite
+////////////////////////////////////////////////////////////////////////////////
+
+function dumpTestSuite () {
```

Comment:
This is just a copy/paste? Because all of those tests here except two are unrelated. Also, lots of tests are commented out fully and not used. We need to clean this up. And re-use same methods please. e.g. expose them to a helper. That duplicate code usage will be so hard to maintain in the future. One change and X-files needs to be adjusted.